### ISSUE TITLE
Fatal error: Uncaught RuntimeException: unknown task: php/early-returns

### ISSUE THREAD

**Issue by ordago** [laravel-shift/workbench-issues#19]
**Desktop:**
 - OS: Windows
 - OS Version: 10
 - App Version:  0.4.0.0

**Describe the bug**
No changes made. Shows error in the log.

**To Reproduce**
1. Run task

```
Fatal error: Uncaught RuntimeException: unknown task: php/early-returns in phar:///opt/shift/workbench/src/Factories/TaskFactory.php:111
Stack trace:
#0 phar:///opt/shift/workbench/src/Shifts/Laravel/WorkbenchBuild.php(40): Shift\Factories\TaskFactory::createTaskFromUses(Array)
#1 [internal function]: Shift\Shifts\Laravel\WorkbenchBuild->Shift\Shifts\Laravel\{closure}(Array, 0)
#2 phar:///opt/shift/workbench/vendor/illuminate/collections/Collection.php(642): array_map(Object(Closure), Array, Array)
#3 phar:///opt/shift/workbench/src/Shifts/Laravel/WorkbenchBuild.php(41): Illuminate\Support\Collection->map(Object(Closure))
#4 [internal function]: Shift\Shifts\Laravel\WorkbenchBuild->Shift\Shifts\Laravel\{closure}(Object(Illuminate\Support\Collection), 'php/early-retur...')
#5 phar:///opt/shift/workbench/vendor/illuminate/collections/Collection.php(642): array_map(Object(Closure), Array, Array)
#6 phar:///opt/shift/workbench/src/Shifts/Laravel/WorkbenchBuild.php(42): Illuminate\Support\Collection->map(Object(Closure))
#7 phar:///opt/shift/workbench/workbench.php(63): Shift\Shifts\Laravel\WorkbenchBuild->__construct(Array)
#8 /opt/shift/workbench(9): require('phar:///opt/shi...')
#9 {main}
  thrown in phar:///opt/shift/workbench/src/Factories/TaskFactory.php on line 111
--------------
```

Unlike the other errors where it would show the error in the log but show the checkmark icon, now it shows the running icon moving. It seems despite throwing a fatal error it managed to shut down docker the docker image.

![image](https://user-images.githubusercontent.com/6376814/125494447-ec271584-0c00-45dd-8241-61b60269a3f1.png)

**Comment by jasonmccreary** [comment]
Pushing a patch for this now.

**Comment by jasonmccreary** [comment]
@ordago, this should be patched in 0.4.1. Keep in mind these are still "Experimental" tasks. So there may still be "bugs" related to their automation. However, the task should no longer throw the "fatal error" reported in this issue.
